azuredeploy.json

```diff
@@ -28,6 +28,20 @@
             "metadata": {
                 "description": "The LORIOT Network Server Management API URL matching to the location of the app."
             }
+        },
+        "sqlAdministratorLogin": {
+            "type": "string",
+            "defaultValue": "",
```

Comment:
The login and password should not have empty defaultValues, because the deployment does not work with empty strings.